aspnetcore/fundamentals/w3c-logger/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: W3C Logger in .NET Core and ASP.NET Core
+author: wtgodbe
+description: Learn how to create server logs in the W3C standard format.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: wigodbe
+ms.custom: mvc
```

Comment:
Don't know what this is for, I copied it from the http-logging doc

Reply:
Referring specifically to `ms.custom`

Reply:
```suggestion
```